src/libraries/System.Net.Http/src/System/Net/Http/Headers/AltSvcHeaderValue.cs

```diff
@@ -13,7 +13,9 @@ namespace System.Net.Http.Headers
     /// </remarks>
     internal sealed class AltSvcHeaderValue
     {
-        public static AltSvcHeaderValue Clear { get; } = new AltSvcHeaderValue("clear", host: null, port: 0, maxAge: TimeSpan.Zero, persist: false);
+        public static string ClearString { get; } = "clear";
```

Comment:
Nit:
```suggestion
        public const string ClearString = "clear";
```
No need respin CI just for this though.